src/_rules/semantic.js

```diff
@@ -0,0 +1,7 @@
+import { handler as h } from '#utils';
+
+export const semanticRules = [
+  [/^s-bg-(.+)$/, ([, cssvar]) => ({ 'background-color': h.semanticToken(`background-${cssvar}`) })],
+  [/^s-text-(.+)$/, ([, cssvar]) => ({ color: h.semanticToken(`text-${cssvar}`) })],
```

Comment:
I saw icon tokens, too. Should they also be added here?

Reply:
Yes, probably... had a question about that in the slack dev-channel. 😄